Src/Engine/Scrabble.php

```diff
@@ -142,10 +142,9 @@ public function matchInDictionary(?string $rack = null): array
      */
     public function removeNonAlphaCharacters(string $string)
     {
-        /**
-         * todo replace this with a method to stip non alpha chars
-         */
-        return $string;
+        //skip non alpha chars
+        $cleanString = preg_replace('/[^a-zA-Z0-9]/', '', $string);
+        return $cleanString;
```

Comment:
no need to create a variable, return directly
```suggestion
        return preg_replace('/[^a-zA-Z0-9]/', '', $string);
```